src/engraving/dom/dynamic.h

```diff
@@ -121,6 +128,7 @@ class Dynamic final : public TextBase
     M_PROPERTY(bool, avoidBarLines, setAvoidBarLines)
     M_PROPERTY(double, dynamicsSize, setDynamicsSize)
     M_PROPERTY(bool, centerOnNotehead, setCenterOnNotehead)
+    M_PROPERTY2(bool, anchorToEndOfPrevious, setAnchorToEndOfPrevious, false)
```

Comment:
I'm not a huge fan of these macros... They complicate navigation in IDE (something like "Find References" doesn't work) and debugging. Also, M_PROPERTY can cause bugs (happened several times already) as it does not initialize the property. Better to avoid them